download.py

```diff
@@ -22,30 +26,87 @@
 if force_automodel:
     print(f"Using AutoModel for {model_name} to instantiate model")
 
-print(f"Downloading model {model_name} from huggingface model hub")
-config = AutoConfig.from_pretrained(model_name)
-model_type = config.to_dict()['model_type']
+onnx_runtime = os.getenv('ONNX_RUNTIME')
+if not onnx_runtime:
+    onnx_runtime = "false"
 
-if model_type is not None and model_type == "t5":
-    SentenceTransformer(model_name, cache_folder=model_dir)
-    with open(f"{model_dir}/model_name", "w") as f:
-        f.write(model_name.replace("/", "_"))
-else:
-    if config.architectures and not force_automodel:
-        print(f"Using class {config.architectures[0]} to load model weights")
-        mod = __import__('transformers', fromlist=[config.architectures[0]])
-        try:
-            klass_architecture = getattr(mod, config.architectures[0])
-            model = klass_architecture.from_pretrained(model_name)
-        except AttributeError:
-            print(f"{config.architectures[0]} not found in transformers, fallback to AutoModel")
-            model = AutoModel.from_pretrained(model_name)
-    else:
-        model = AutoModel.from_pretrained(model_name)
+onnx_cpu_arch = os.getenv('ONNX_CPU')
+if not onnx_cpu_arch:
+    onnx_cpu_arch = "arm64"
+
+print(f"Downloading MODEL_NAME={model_name} with FORCE_AUTOMODEL={force_automodel} ONNX_RUNTIME={onnx_runtime} ONNX_CPU={onnx_cpu_arch}")
+
+def download_onnx_model(model_name: str, model_dir: str):
+    # Download model and tokenizer
+    onnx_path = Path(model_dir)
+    ort_model = ORTModelForFeatureExtraction.from_pretrained(model_name, from_transformers=True)
```

Comment:
I'm surprised that this is the `optimum.onnxruntime` class to use for loading embedding models with ONNX but it seems to be used elsewhere for the same task